src/GraphQL.Tests/Utilities/SchemaBuilderTestBase.cs

```diff
@@ -23,7 +23,17 @@ public ExecutionResult AssertQuery(Action<ExecuteConfig> configure)
         configure(config);
 
         var schema = Builder.Build(config.Definitions);
+
+        return AssertQuery(configure, schema);
```

Comment:
This calls the other overload, which runs ```configure(config)``` a second time -- ???

Reply:
Yeah, it is not really clean, but the problem is, that the overloading methods already need the configured config, but need to pass on the configure Action. In the existing code this was already implicitly the case with the existing Overloads. (Slightly hidden in the `var runResult = Executer.ExecuteAsync(options).Result;` line). (So actually at the moment the configure Action is called three times :sweat_smile: )

Reply:
Not too worried about how the tests run so that’s fine.